README.md

```diff
@@ -5,6 +5,8 @@ This repository contains the Ensign driver, SDK, and helpers for Python. For the
 
 ## Installation
 
+PyEnsign is compatible with Python 3.4 or later. The simplest way to install PyEnsign and its dependencies is from PyPI with pip, Python's preferred package installer.
```

Comment:
gRPC python (the `grpcio` package) is apparently only supported for Python 3.7 and later, so I suppose that means pyensign is also >= 3.7